src/main/java/com/gluonhq/richtext/model/TextDecoration.java

```diff
@@ -2,21 +2,24 @@
 
 import javafx.scene.paint.Color;
 import javafx.scene.text.Font;
-import javafx.scene.text.Text;
+import javafx.scene.text.FontPosture;
+import javafx.scene.text.FontWeight;
 
-import java.util.Optional;
+import java.util.Objects;
 
 public class TextDecoration {
 
     public static TextDecoration DEFAULT = builder()
             .foreground(Color.BLUE)
```

Comment:
Shouldn't be needed if builder defines some default colors

Reply:
Done